.github/workflows/link-to-checklist.yml

```diff
@@ -0,0 +1,31 @@
+name: link-to-checklist
+
+on:
+  workflow_dispatch:
+  pull_request_target:
+    types: [opened]
+
+# permissions:
+#   pull-requests: write
+
+jobs:
+  add-review-checklist:
+    if : ${{ github.event_name == 'pull_request' && github.event.action == 'opened' && startsWith(github.event.pull_request.title, 'ingestion') == true}}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v3
+      - name: Add review checklist
+        uses: actions/github-script@v6
+        with:
+          # github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            const fs = require('fs');
+            const review_checklist = fs.readFileSync('.github/ingestion-review-checklist.md', 'utf8');
+
+            github.rest.issues.createComment({
+              issue_number: context.issue.number,
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              body: review_checklist,
```

Comment:
It looks like this creates a comment on the PR. This could work and is maybe a good starting point, but I wonder if there is a way to have the checklist as the PR's description?

Reply:
I think it's cleaner to have it as a comment, as otherwise we have to deal with adding the checklist to the pr.  Github has all the needed functionality in their [pr rest api](https://docs.github.com/en/rest/pulls/pulls?apiVersion=2022-11-28#update-a-pull-request); I am not sure whether it's worth the effort to implement it right now.